packs/vsphere/actions/vmwarelib/actions.py

```diff
@@ -9,13 +9,16 @@
 class BaseAction(Action):
     def __init__(self, config):
         super(BaseAction, self).__init__(config)
-        self.service_instance = self._connect()
+        self.si = self._connect()
+        self.si_content = self.si.RetrieveContent()
+        #service_instance is actually content here. Leaving for legacy.
+        self.service_instance = self.si.RetrieveContent()
```

Comment:
si.service_instance = self.si_content. 

I assume RetrieveContent is a remote call?